scipy/special/orthogonal.py

```diff
@@ -1386,6 +1388,30 @@ def gegenbauer(n, alpha, monic=False):
     :math:`[-1,1]` with weight function :math:`(1 - x^2)^{(\alpha -
     1/2)}`.
 
+    Examples
+    --------
+    >>> from scipy import special
+    >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
```

Comment:
This `import numpy as np` is always assumed in all SciPy code examples, and doesn't need to be added.